inst/templates/Makefile

```diff
@@ -0,0 +1,29 @@
+R = R --vanilla CMD
+
+
+all: build check install clean
+
+build:
+	$(R) build .
```

Comment:
I guess we should think about if it would be better to build the tarball in the current working directory or the parent... I don't have a strong preference either way...

Reply:
Hmmm, good point. I feel that if it's built in the current directory (hopefully package top-level), then that would be easier to control for by the end-user (via .gitignore etc), and more portable besides. If a user stores all their package code in folders where the parent of each is write-restricted or something else weird, then the build step would fail. Enforcing focus on the current-and-below path should be much more manageable, by everyone. I'm happy to hear thoughts, though.